test/test_profiler.py

```diff
@@ -137,7 +137,7 @@ def test_kineto(self):
         for e in p.function_events:
             if "aten::mm" in e.name:
                 found_mm = True
-            if "gemm" in e.name:
+            if "gemm" in e.name or "Cijk" in e.name:
```

Comment:
@mwootton @jeffdaily I think this should be cherry-picked into Michael's Kineto branch to make sure we include this in the upstreaming for Kineto